ML/AliMLModelHandler.cxx

```diff
@@ -153,6 +154,33 @@ std::string AliMLModelHandler::ImportFile(std::string path) {
     }
     return path;
   }
+
+  // check if file is on cvmfs
+  if (path.find("/cvmfs") != std::string::npos) {
+    bool checkFile = gSystem->AccessPathName(path.c_str());
+    if (checkFile) {
+      AliFatalClass(Form("Error file %s not found on CVMFS! Exit", path.data()));
+    }
+    // only skip copying for certain file types
+    std::string fileform = path.substr(path.find_last_of(".")+1,path.size());
+    if(fileform == "so" || fileform == "yml" || fileform ==  "yaml"){
+      return path;
+    }
+  }
```

Comment:
I discussed a bit with @fgrosa which explained to me that this is intended to allow users to access the data on cvmfs locally using their installation of AliPhyscis. 
However, from my understanding, this won't work because locally you can access only data exporting `ALICE_DATA=root://eospublic.cern.ch//eos/experiment/alice/analysis-data' and reading it using ROOT methods.

Reply:
Hi @fcatalan92, indeed, from a local installation this is not possible, because as you write it should use ROOT methods. However this method is intended for lxplus (or machines like aliceml) where one can access directly the files via a `/cvmfs` path

Reply:
Hi Luuk, what Fabio and I realised is that if you run it loading AliPhysics from `cvmfs`, it works from scratch because it is covered by the next if statement (since `AliDataFile::GetFileName()` returns the `cvmfs` path if you do so), so indeed we can avoid to have this additional if.

Reply:
Hi Fabrizio, sorry I should have clarified better. This part should be seen as a safety if the user provides the full cvmfs path (for some reason). Indeed the part with `AliDataFile` is the "main" part. So this can be removed if you think it is not worth to keep this as safety

Reply:
Ok I got it now, so I have no strong opinion on this, indeed if you think that someone could set directly the `cvmfs` path we can also keep it, even if maybe in general it would be better to let `AliDataFile::GetFileName()` getting the correct path.

Reply:
I'm for supporting only the case with `AliDataFile::GetFileName()`, but it is not a strong opinion.

Reply:
I prefer having the possibility of specifying the path manually even from cvmfs